src/clang-c-frontend/clang_c_adjust.h

```diff
@@ -42,6 +42,7 @@ class clang_c_adjust
   void adjust_symbol(exprt &expr);
   void adjust_comma(exprt &expr);
   void adjust_builtin_va_arg(exprt &expr);
+  virtual void adjust_decl_block(codet &code);
```

Comment:
Since this is a virtual method, could you add a description for it?

Reply:
Done. Added a comment.